oe_corporate_blocks.permissions.yml

```diff
@@ -0,0 +1,6 @@
+administer footer link general:
+  title: 'Administer footer link general'
```

Comment:
We can have only one permission that deals with both types of link, let's call it:
```
administer site specific footer links:
  title: 'Administer site specific footer links'
```